util/file.go

```diff
@@ -94,4 +102,12 @@ func ReadFileAsString(path string) (string, error) {
 	}
 
 	return string(bytes), nil
+}
+
+func JoinPath(elem ...string) string {
+	return filepath.ToSlash(filepath.Join(elem...))
+}
+
+func CleanPath(path string) string {
```

Comment:
Could you add comments to these two methods explaining what they do and why the should be used (i.e. for cross-OS compatibility).